spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2889,6 +2878,35 @@ object QueryPlanSerde extends Logging with CometExprShim {
     }
   }
 
+  /**
+   * Determine which data types are supported in a shuffle.
+   */
+  def supportedShuffleDataType(dt: DataType): Boolean = dt match {
+    case _: BooleanType => true
```

Comment:
nit : BooleanType moved here alone because of the code style checks?

Reply:
No. At one point I was seeing errors related to boolean and had made functional changes here that I later reverted.

Reply:
I reverted the style change